pages/app-layout/dashboard-content-type.page.tsx

```diff
@@ -33,6 +42,7 @@ export default function () {
           </SpaceBetween>
         }
         splitPanelPreferences={{ position: 'side' }}
+        onSplitPanelPreferencesChange={() => {}}
```

Comment:
Fixed a development warning that `splitPanelPreferences` cannot be used without change callback